sub_workflows/kfdrc_lancet_sub_wf.cwl

```diff
@@ -17,16 +18,30 @@ inputs:
   bed_invtl_split: {type: 'File[]', doc: "Bed file intervals passed on from and outside pre-processing step"}
   ram: {type: ['null', int], default: 12, doc: "Adjust in rare circumstances in which 12 GB is not enough."}
   select_vars_mode: {type: ['null', {type: enum, name: select_vars_mode, symbols: ["gatk", "grep"]}], doc: "Choose 'gatk' for SelectVariants tool, or 'grep' for grep expression", default: "gatk"}
-  vep_cache: {type: File, label: tar gzipped cache from ensembl/local converted cache}
+  vep_cache: {type: File, doc: "tar gzipped cache from ensembl/local converted cache"}
   vep_ref_build: {type: ['null', string], doc: "Genome ref build used, should line up with cache.", default: "GRCh38" }
   window: {type: int, doc: "window size for lancet.  default is 600, recommend 500 for WGS, 600 for exome+"}
-  padding: {type: int, doc: "If WGS (less likely), default 25, if exome+, recommend half window size"}
+  padding: {type: int, doc: "If WGS (less likely), default 25, if exome+, recommend half window size, 0 if pure exome or panel"}
+  tool_name: {type: string?, doc: "String to describe what tool was run as part of file name", default: "lancet_somatic"}
+  # annotation vars
+  genomic_hotspots: { type: 'File[]?', doc: "Tab-delimited BED formatted file(s) containing hg38 genomic positions corresponding to hotspots" }
+  protein_snv_hotspots: { type: 'File[]?', doc: "Column-name-containing, tab-delimited file(s) containing protein names and amino acid positions corresponding to hotspots" }
+  protein_indel_hotspots: { type: 'File[]?', doc: "Column-name-containing, tab-delimited file(s) containing protein names and amino acid position ranges corresponding to hotspots" }
+  retain_info: {type: string?, doc: "csv string with INFO fields that you want to keep", default: "MS,FETS,HotSpotAllele"}
+  retain_fmt: {type: string?, doc: "csv string with FORMAT fields that you want to keep"}
+  maf_center: {type: string?, doc: "Sequencing center of variant called", default: "."}
+  add_common_fields: {type: boolean?, doc: "Set to true if input is a strelka2 vcf that hasn't had common fields added", default: false}
```

Comment:
What input might be a strelka2 vcf?

Reply:
Ah, it's kind of copy/paste as it gets fed into the annotation subworkflow. So, this is generic, but I do set the default to fit this case